### ISSUE TITLE
Download link on https://github.com/cpp-netlib/cpp-netlib is not up to date

### ISSUE THREAD

**Issue by hlovdal** [cpp-netlib/cpp-netlib#672]
At https://github.com/cpp-netlib/cpp-netlib it says

> You can find official release packages of the library at:
> http://github.com/cpp-netlib/cpp-netlib/downloads

but that page does not contain releases newer than 0.9.4 while the current release is 0.12.0.

So either update http://github.com/cpp-netlib/cpp-netlib/downloads to include newer releases or update the download link.

**Comment by deanberris** [comment]
You're right -- we host the links on the website now (http://cpp-netlib.org/) from where you can get the latest downloads. This documentation for the project is included in the `README.rst` file at the root of the project directory.

Note that we don't allow listing all the packages available for download (they're hosted on Google Cloud Storage), but we try and regularly update the website whenever there's a new release out.

I think we should just point to the official website and let users download the officially-released tar-balls from there.

What do you think?

**Comment by hlovdal** [comment]
Linking to the official website for downloads is perfectly fine I think, although I would like there to be a simple way to download older releases for those times when that is what you want. Having this "hidden away" on a separate _old releases_ page is fine.

When someone tries to compile a program with an older version of cpp-netlib as dependency (and which does not compile with the latest version), then it is frustrating if the project website does not have those available.

I just needed to [compile with version 0.11.2](https://github.com/performous/performous/issues/201), and copying and changing the link from http://downloads.cpp-netlib.org/0.12.0/cpp-netlib-0.12.0-final.tar.bz2 to http://downloads.cpp-netlib.org/0.11.2/cpp-netlib-0.11.2-final.tar.bz2 worked fine, but that was just guessing and luck on my part, it is not a solution that scales since for instance http://downloads.cpp-netlib.org/0.9.4/cpp-netlib-0.9.4-final.tar.bz2 is not a valid link. So please do provide access to older releases.

**Comment by deanberris** [comment]
Good points -- we should make the following two changes:

- Make a change to the website so we have a list of all the previous releases and how to get them.
- Host all the downloadable binaries in Google Cloud Storage, and use a sane/consistent naming scheme for the URLs so those could be deduced mechanically (like how we have it now).

I don't think I can get to it anytime soon (been swamped recently with other things) but I do think this is important to address.
